cmake/FindFuthark.cmake

```diff
@@ -0,0 +1,125 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2021-2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# The C language is not enabled in our project by default, but we need it to
+# compile Futhark's code.
+enable_language(C)
```

Comment:
"Find modules" are not meant to make a CMake configuration fail themselves. They can instruct their caller to bail out because they didn't find what they were looking for, but they shouldn't have the equivalent of `exit(1)` ever.

Long story short, this should be replaced by:

```cmake
include(CheckLanguage)
check_language(C)
if(CMAKE_C_COMPILER)
   enable_language(C)
endif()
```

And then `CMAKE_C_COMPILER` needs to be a "required variable" for this module "to be found". (More on that later on.)